common/python_health_helper.sh

```diff
@@ -4,7 +4,7 @@ usage() {
   if [ -n "$1" ]; then
```

Comment:
Can you log an issue to rewrite this shell script in Python?

Reply:
Will do. But I don't see particular value in that exercise.